install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Mac OS Detected"
+    export OS="MAC"
+    export MINICONDA_SCRIPT=Miniconda2-latest-MacOSX-x86_64.sh
+    # Mac OS X, CPU only, Python 2.7:
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-0.11.0rc0-py2-none-any.whl
+elif [ "$(uname)" == "Linux" ]; then
+    echo "Linux OS Detected"
+    export OS="LINUX"
+    export MINICONDA_SCRIPT=Miniconda2-latest-Linux-x86_64.sh
+    # Ubuntu/Linux 64-bit, CPU only, Python 2.7
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc0-cp27-none-linux_x86_64.whl
+fi
+
+
+# Check if anaconda already installed
+if [ ! $(which conda) ]; then
+    echo ""
+    echo "==========================================="
+    echo "anaconda not detected, installing miniconda"
+    echo "==========================================="
+    echo ""
+    curl https://repo.continuum.io/miniconda/$MINICONDA_SCRIPT > $/tmp/$MINICONDA_SCRIPT
```

Comment:
should quote these strings: https://google.github.io/styleguide/shell.xml?showone=Constants_and_Environment_Variable_Names#Quoting